resources/home/dnanexus/utils/util_functions.py

```
@@ -0,0 +1,101 @@
+            file_data = sample_data["clinical_indications"][clin_ind]
+
+            # If variant count was never present then replace with None
+            # so we can easily not include this field in final .xlsx
+            # If snv_reports option True, replace SNV report URL with text
+            # if variant count is zero.
+            if "SNV" in file_data:
+                for file in file_data["SNV"]:
+                    if file.get("SNV count") == "Unknown":
+                        file["SNV count"] = None
+                    if snv_reports:
+                        if file.get("SNV count") == "0":
```

Ruff (0.8.0)

Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)